app/src/main/java/br/com/dionataferraz/vendas/account/usecase/findAccountUseCase.kt

```diff
@@ -4,14 +4,18 @@ import br.com.dionataferraz.vendas.App
 import br.com.dionataferraz.vendas.account.data.local.AccountDatabase
 import br.com.dionataferraz.vendas.account.data.local.AccountEntity
 
-class findAccountUseCase(private val id: Int) {
+class findAccountUseCase() {
```

Comment:
Como é uma classe ela deve iniciar como upper case 
```suggestion
class FindAccountUseCase() {
```